clients/go/compat.go

```diff
@@ -57,6 +63,7 @@ func PackTx(tx types.Transaction, cipher Cipher) (*types.Transaction, error) {
 }
 
 func packTx(tx types.Transaction, cipher Cipher) (*types.Transaction, error) {
+	fmt.Printf("NONCE: %d\n", tx.Nonce())
```

Comment:
(nit maybe) Do we always want to log?

Reply:
This should be removed, likely a debugging leftover :)